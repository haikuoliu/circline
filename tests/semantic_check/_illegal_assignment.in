int a = 3.0;