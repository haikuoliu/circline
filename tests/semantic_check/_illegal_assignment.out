illegal assignment int = float in a = 3..
